python/hail/methods/statgen.py

```diff
@@ -2291,12 +2246,13 @@ def __init__(self, filter_expr, keep=True, left_aligned=False, keep_star=False):
 
         self._old_to_new = construct_reference('oldToNew', TArray(TInt32()), ds._row_indices)
         self._new_to_old = construct_reference('newToOld', TArray(TInt32()), ds._row_indices)
-        self._new_v = construct_reference('newV', ds._rowkey_schema, ds._row_indices)
+        self._new_locus = construct_reference('newLocus', ds['locus'].dtype, ds._row_indices)
+        self._new_alleles = construct_reference('newAlleles', ds['alleles'].dtype, ds._row_indices)
 
     @property
     def new_to_old(self):
         """The array of old allele indices, such that ``new_to_old[newIndex] =
-        oldIndex`` and ``newToOld[0] = 0``.  A row-indexed expression.
+        oldIndex`` and ``newToOld[0] = 0``. A row-indexed expression.
```

Comment:
should be `new_to_old[0] = 0`.  Also, maybe double equal signs?

Reply:
hey, you wrote that! (and I failed to properly review it)